### ISSUE TITLE
Insufficient Color Contrast in Dark Mode

### ISSUE THREAD

**Issue by sofistonehouse** [gordon-cs/gordon-360-ui#2136]
Gordon-blue against dark background doesn't have enough contrast to pass accessibility tests in Lighthouse. This is present throughout 360. 

To Complete:
- #2270 
- #2311 
- ...?
